package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-activity-monitor",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
This is a fairly substantial change (new feature). I think it warrants a minor bump (instead of patch level) :)